packages/ni.datamonikers.v1.proto/pyproject.toml

```
@@ -0,0 +1,96 @@
+[tool.poetry]
+name = "ni.datamonikers.v1.proto"
+version = "0.1.0-dev1"
+license = "MIT"
+description = "Protobuf data types and service stub for NI moniker gRPC APIs"
+authors = ["NI <opensource@ni.com>"]
+maintainers = [
+  "Brad Keryan <brad.keryan@emerson.com>",
+  "Joe Friedrichsen <joe.friedrichsen@emerson.com>",
+]
+readme = "README.md"
+repository = "https://github.com/ni/ni-apis-python"
+keywords = ["ni-apis", "moniker"]
```

Keywords are [basically for SEO](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#keywords). Is "moniker" a useful search term? https://pypi.org/search/?q=moniker

Would "data-moniker" be more appropriate for NI customers?
```suggestion
keywords = ["ni-apis", "data-moniker"]
```

---

Seems reasonable.